### ISSUE TITLE
Seemingly unable to plain 'require mro'

### ISSUE THREAD

**Issue by ribasushi** [perl11/cperl#125]
Hi! Could you please shed some light on how I can solve issues [like this one](http://www.cpantesters.org/cpan/report/5354fc98-f581-11e5-83f7-e08840963b06)? I am [not asking for any kind of version](https://metacpan.org/source/RIBASUSHI/Class-Accessor-Grouped-0.10012/lib/Class/Accessor/Grouped.pm#L14) myself, seems to be something within cperl itself. Let me know how I can help / what should I change.

Cheers!

**Comment by rurban** [comment]
Ah, you can ignore that, sorry. 
My $mro::VERSION still has a 'c' suffix. I thought I already got rid of them already at run-time.

**Comment by ribasushi** [comment]
@rurban This is [still a problem on recent smokes](http://www.cpantesters.org/cpan/report/2a1e76a0-fc94-11e5-833e-a5b240963b06). What is the ETA on fixing this?

**Comment by rurban** [comment]
I'll check ASAP. This is from the sigs branch, which is highly unstable. I'll first fix a Moose crash.

**Comment by rurban** [comment]
I see now what's going on. I cannot mix libs with signatures with a binary without and vice versa.
It's an entirely new architecture.
All signatured "modern" libs need to go into archlib, not into privlib, or I need a major version bump.

Fixed with 
commit f840fd8edd2fbe4c2cdaf2056b3bb0a8b9605fe6
Author: Reini Urban rurban@cpanel.net
Date:   Sat Apr 9 09:44:41 2016 +0200

```
installperl: modernized_cperl modules into archlib

do not install them into privlib for this 5.22.2 series, until it is merged.
we cannot load modernized modules into branches without sigs

This also fixes #125, wrong CPAN::Reporter reports with sig and VERSION mixups.
```

I'll disable CPAN::Reporter for a signatured cperl for now.
